src/components/radio-button/radio-button.js

```diff
@@ -30,14 +31,32 @@ import { tagComponent } from '../../utils/helpers/tags';
  */
 const RadioButton = Input(InputLabel(InputValidation(
 class RadioButton extends React.Component {
+  static propTypes = {
+    /**
+     * A custom class name for the component.
+     *
+     * @property className
+     * @type {String}
+     */
+    className: PropTypes.string,
```

Comment:
I wonder due to the safeProps thing if defining className here will stop it being applied correctly. I don't think in general we need to be defining these kind of prop types that are always available